### DISCUSSION TITLE
New questions about buildings

### DISCUSSION THREAD

**Discussion by BLINMATIC** [streetcomplete/StreetComplete#5484] [Ideas]
It would be nice if we could answer these in the app
1. How many floors does the building have?
2. How tall is the building?
3. How many units does the building have?

**Comment by matkoniecz** [comment]
> How many units does the building have?

Kind of indirectly asked by entrance quest (requires mapping of footways/steps to reach places where entrance is in some other editor)

> How tall is the building?

Is it passing https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md ?

> How many floors does the building have?

This is already available - just disabled by default. You can enable extra quests in settings.

**Reply by BLINMATIC** [reply]
`How tall is the building?`

Yes I think it passes since it is just a number to be entered and the height can be measured by a phone with a camera and software. And in some cases you just need to multiply the floor count by 3. It is kind of similar to the `How wide is this road?` question.

**Reply by matkoniecz** [reply]
> And in some cases you just need to multiply the floor count by 3

In such case estimation can be done using already floor count (and I would not put `height` tag based on assumption that every floor takes exactly 3m)

**Reply by BLINMATIC** [reply]
yes but it can differ on some buildings as some buildings have decorative parts that don't count as a floor.

**Reply by Helium314** [reply]
Even with camera and great software, in cities the height is very hard to measure, as the top edge of the roof might simply not be visible from the road.
